### PR TITLE
build: offload hash checking logic to lambda worker during release

### PR DESCRIPTION
**Pull request by MarshallOfSound**
During the sudowoodo release phase we download all the release assets and manually verify their hashes.  Previously this happened via the `sumchecker` module and downloading every asset in parallel onto the local heroku node.

This was terribly slow, although our assets only sum to around ~10GB the "validate assets" step took almost 20 minutes this morning.  It's slow for a few reasons but mostly (a) we're severely bandwidth limited, the download speed on heroku isn't that fast and (b) we're disk / CPU limited, heroku's I/O performance isn't great either.

I spun up a quick lambda function in our AWS account that takes a URL and a hashing algorithm and _very_ quickly returns the hash.  I managed to validate the checksums of every release asset in the beta 32 release (that went out this morning) in just under 37 seconds.

TLDR: Lambda go brrr

Notes: no-notes

### CODE REVIEW

File: script/release/get-url-hash.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  return new Promise((resolve, reject) => {
+    lambda.invoke({
+      FunctionName: 'hasher',
+      Payload: JSON.stringify({
+        targetUrl,
+        algorithm
+      })
+    }, (err, data) => {
+      if (err) return reject(err);
+      let hash;
```
Comment (codebytere) [inline script/release/get-url-hash.js:21]: why do we need this here? can we just resolve `response.hash` or reject in the try/catch?

File: script/release/release.js [hunk 2]
```diff
@@ -391,137 +396,59 @@ async function verifyAssets (release) {
...
-    await validateChecksums({
-      algorithm: 'sha1',
-      filesToCheck,
-      fileDirectory: downloadDir,
-      shaSumFile: 'SHASUMS.txt',
-      checkerOpts,
-      fileSource
-    });
+async function validateFileHashesAgainstShaSumMapping (remoteFilesWithHashes, mapping) {
+  for (const remoteFileWitHash of remoteFilesWithHashes) {
```
Comment (codebytere) [inline script/release/release.js:419]: tiny typo s/remoteFileWitHash/remoteFileWithHash

Comment (codebytere) [inline script/release/release.js:419]: tiny typo s/remoteFileWitHash/remoteFileWithHash

File: script/release/get-url-hash.js (hunk 1 above)
Comment (codebytere) [inline script/release/get-url-hash.js:21]: why do we need this here? can we just resolve response.hash or reject in the try/catch?

File: script/release/release.js [hunk 3]
```diff
@@ -5,20 +5,16 @@ if (!process.env.CI) require('dotenv-safe').load();
 const args = require('minimist')(process.argv.slice(2), {
   boolean: [
     'validateRelease',
-    'skipVersionCheck',
```
Comment (codebytere) [inline script/release/release.js:8]: is this just unused? we also pass this to Sudowoodo - we should delete if thats the case
